robe.el

```diff
@@ -83,6 +83,25 @@ have constants, methods and arguments highlighted in color."
 
 (defvar robe-running nil)
 
+(defcustom robe-completing-read-func 'ido-completing-read
+  "Function to call for completing read. Will not be used when either
```

Comment:
> The first line of the documentation string should consist of one or two complete sentences that stand on their own as a summary.

http://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html

And since you're saying "either", how can `completing-read` be overridden, aside from using `completing-read-function`?